src/test/java/io/federecio/dropwizard/swagger/selenium/auth/DefaultServerWithAuthenticationSeleniumTest.java

```diff
@@ -17,6 +17,7 @@
 import java.util.List;
 import java.util.concurrent.TimeUnit;
 import org.junit.ClassRule;
+import org.junit.Ignore;
```

Comment:
I need to update the readme on what is required to execute the tests with this.  They run for me ok after installing https://sites.google.com/a/chromium.org/chromedriver/ and having the latest Chrome version.